src/lib.rs

```diff
@@ -236,13 +236,19 @@ pub struct CommandBuffer {
 #[derive(Debug)]
 pub struct CommandEncoder {
     id: wgc::id::CommandEncoderId,
+    /// This type should be !Send !Sync, because it represents an allocation on this thread's
+    /// command buffer.
+    _p: std::marker::PhantomData<*const u8>,
 }
 
 /// An in-progress recording of a render pass.
 #[derive(Debug)]
 pub struct RenderPass<'a> {
     id: wgc::id::RenderPassId,
     _parent: &'a mut CommandEncoder,
+    /// This type should be !Send !Sync, because it represents an allocation on this thread's
+    /// command buffer.
+    _p: std::marker::PhantomData<*const u8>,
```

Comment:
is this needed? I thought, since it borrows the encoder, it has the same restrictions

Reply:
I'll remove that, it's not strictly necessary yea